<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hammer 5 Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/appicon.ico</normaloff>:/icons/appicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QTabWidget" name="MainWindowTools_tabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #1d1d1f;
QTabWidget::pane {
    background-color: solid red;
    border-radius: 0px;
    border: 2px solid gray;
    border-color: #363639;
    background-color: #1d1d1f;
}
/* ========================================================== */
QTabBar::tab {
    background-color: #323232;
    color: #9A9F91;
    border-radius: 0px;
    border-top-right-radius: 0px;
    border-top-left-radius: 0px;
    padding: 4px;
    padding-left:48px;
    padding-right: 48px;

    border-top: 2px solid gray;
    border-bottom: 0px solid black;

    font: 700 10pt &quot;Segoe UI&quot;;
    border-left: 2px solid darkgray;
    border-top: 0px solid darkgray;
    border-color: #151515;
    border-right: 2px solid rgba(80, 80, 80, 80);



    color: #E3E3E3;
    background-color: #151515;

}
QTabBar::tab:selected {
    border-radius: 0px;
    border-top-right-radius: 7px;
    border-top-left-radius: 7px;

    border-top: 2px solid gray;
    border-left: 2px solid gray;
    border-right: 2px solid gray;
    border-bottom: 0px solid black;

    font: 700 10pt &quot;Segoe UI&quot;;
    border-color: rgba(80, 80, 80, 180);
    height:20px;
    color: #E3E3E3;
    background-color: #1d1d1f;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Loading_Editor_Tab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/imagesmode_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/imagesmode_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Loading Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QWidget" name="soundeditor_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/volume_up.png</normaloff>:/icons/volume_up.png</iconset>
       </attribute>
       <attribute name="title">
        <string>SoundEvent Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </widget>
      <widget class="QWidget" name="smartpropeditor_tab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/functions_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/icons/functions_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </attribute>
       <attribute name="title">
        <string>SmartProp Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </widget>
      <widget class="QWidget" name="BatchCreator_tab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/dashboard_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</normaloff>:/icons/dashboard_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.png</iconset>
       </attribute>
       <attribute name="title">
        <string>AssetGroup Maker</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
      <widget class="QWidget" name="hotkeyeditor_tab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/keyboard.png</normaloff>:/icons/keyboard.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Hotkey Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="my_twitter_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777210</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Twitter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/social/icons/social/twitter.png</normaloff>:/social/icons/social/twitter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="discord">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777210</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Discord server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/social/icons/social/discord.png</normaloff>:/social/icons/social/discord.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="documentation_button">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777210</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Discord server&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/help_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/help_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FixNoSteamLogon_Button">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Fix No Steam Logon&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This script restarts Steam and launches the current addon.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;
font: 580 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Restart</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/valve_common/icons/tools/common/valve_small.png</normaloff>:/valve_common/icons/tools/common/valve_small.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="preferences_button">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;
font: 580 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/settings_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/settings_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>24</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="export_and_import_addon_button">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Archive addon&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Create an archive on from game and content folders.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/publish_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/publish_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="delete_addon_button">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Delete addon&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Deletes selected addon.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/delete_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/delete_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="create_new_addon_button">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Create addon&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Quick create an addon with custom presets.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/post_add_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/post_add_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboBoxSelectAddon">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Addon selection&lt;/span&gt;&lt;/p&gt;&lt;p&gt;In the Addon selection, all tools apply changes to the addon selected in this section.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="launch_settings">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/settings_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/settings_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Launch_Addon_Button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Addon launch&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Launches the current addon selected in the addon selection.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;
font: 700 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Edit map</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/hammer_icon.png</normaloff>:/icons/icons/hammer_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>24</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="open_addons_folder_button">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Open Game/Content Addons folder&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Opens the game addons or content addons folder for the current addon selection. &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;To choose the folder type to open, select a folder in the &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;Addon Folder Selection&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/folder_open.svg</normaloff>:/icons/folder_open.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="open_addons_folder_downlist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Addon Folder Selection&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Used to choose the folder that should be oppened in the &lt;span style=&quot; font-style:italic;&quot;&gt;Open Game/Content Addons Folder.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 580 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Open Folder</string>
        </property>
        <item>
         <property name="text">
          <string>Content</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Game</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanup_button">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Cleanup&lt;/span&gt;&lt;/p&gt;&lt;p&gt;...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 5px;
font: 580 10pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Cleanup</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/valve_common/icons/tools/common/find.png</normaloff>:/valve_common/icons/tools/common/find.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
